spread.yaml

```diff
@@ -848,14 +848,15 @@ suites:
             distro_update_package_db
             distro_install_package snapd qemu qemu-utils genisoimage sshpass qemu-kvm cloud-image-utils ovmf kpartx xz-utils
             snap install ubuntu-image --classic
-
-            # TODO: replace with distro package once 2.45 is in stable
-            dpkg -i "$SPREAD_PATH"/../snapd_*.deb
```

Comment:
hmm why is it desirable to just use snapd from the distro instead of the one we build for the spread run?

for example the distro package of snapd will not trust the testkeys which I am currently writing tests for, so if we do go back to using snapd from distro for the host VM, my test will need to install the snapd from the spread run and then uninstall it which is awkward and feels wrong

Reply:
Nice, so I reverted that and installed snapd in all the nested suites.
Thanks for the review